site/kubernetes/operator/kubectl-plugin.md

```diff
@@ -0,0 +1,211 @@
+
+<pre class="lang-bash">
+kubectl rabbitmq list
+</pre>
+
+Displays all RabbitMQ clusters deployed by the RabbitMQ cluster operator on the target kuberentes instance.
+
+### <a id='delete-rmq' class='anchor' href='#delete-plugin'>Delete RabbitMQ cluster</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq delete INSTANCE
+</pre>
+
+Delete a RabbitMQ cluster, or multiple RabbitMQ clusters. When deleting multiple RabbitMQ clusters, provide a space
+seperated list.
+
+<pre class="lang-bash">
+kubectl rabbitmq delete rmq1 rmq2 rmq3
+</pre>
+
+### <a id='secrets' class='anchor' href='#secrets'>Print default user secrets</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq secrets INSTANCE
+</pre>
+
+Displays the default user secrets for the named RabbitMQ cluster
+
+### <a id='manage' class='anchor' href='#manage'>Open the Management UI</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq manage INSTANCE
+</pre>
+
+Opens the RabbitMQ Management dashboard in a browser.
+
+### <a id='debug' class='anchor' href='#debug'>Set Log Level to debug</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq debug INSTANCE
+</pre>
+
+Sets the log level on all nodes to debug. For a detailed breakdown on RabbitMQ logging
+see the [logging docs page](/logging.html).
+
+### <a id='tail' class='anchor' href='#tail'>Tail Logs</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq tail INSTANCE
+</pre>
+
+Attach to live log output from all RabbitMQ nodes.
+
+### <a id='observe' class='anchor' href='#observe'>Observe</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq observe INSTANCE INDEX
+</pre>
+
+Opens the rabbitmq-diagnostics observer interface for a given node. For more
+information on the rabbitmq-diagnostics see [here](rabbitmq-diagnostics.8.html).
+
+### <a id='feature-flags' class='anchor' href='#feature-flags'>Enable All Feature Flags</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq enable-all-feature-flags INSTANCE 
+</pre>
+
+This command wraps the `rabbitmqctl` CLI to enable all possible feature flags.
+For comprehensive documentation on all feature flags see [here](/feature-flags.html).
+
+### <a id='pause-reconciliation' class='anchor' href='#pause-reconciliation'>Pause Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq pause-reconciliation INSTANCE 
+</pre>
+
+This adds the label "rabbitmq.com/pauseReconciliation=true" to your RabbitMQ cluster, which prevents the operator from
+watching and updating the instance.
+
+### <a id='resume-reconciliation' class='anchor' href='#resume-reconciliation'>Resume Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq resume-reconciliation INSTANCE 
+</pre>
+
+Instructs the operator to resume reconciliation for a RabbitMQ cluster.
+
+### <a id='list-pause-reconciliation-instances' class='anchor' href='#list-pause-reconciliation-instances'>List Instances with Paused Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq resume-reconciliation INSTANCE 
```

Comment:
This looks like a copy-paste artefacts, the command here is probably not `resume-reconciliation` but something else?

Reply:
Ah yes, good catch!